Jenkinsfile

```diff
@@ -1144,6 +1146,7 @@ pipeline {
             when {
                 beforeAgent true
                 // expression { skipTest != true }
+                expression { env.NO_CI_TESTING != 'true' }
```

Comment:
Probably want to wrap these two expressions in an `allOf { }` wrapper.

Reply:
I'm still a fan of explicit grouping vs. relying on implicit/default behaviour, if just for clarity.